src/core/text.ts

```diff
@@ -179,6 +181,15 @@ export const login = async ({
       subLogger.info({ phone }, "user logged in")
     } else {
       user = await User.findOneAndUpdate({ phone }, {}, { upsert: true, new: true })
+      const res = await Authorization.initAccountPermissions({
```

Comment:
Couldn't find the implementation for this, is it uncommitted?

Reply:
You're right I had forgotten to commit it - look here https://github.com/GaloyMoney/galoy/blob/3cbd7760485d285b493e0f42ad1fd4ba5e1a6a49/src/app/authorization/index.ts#L10